README.md

```diff
@@ -167,6 +163,30 @@ py.test --numprocesses=4 --looponfail --maxfail=1
 pytest -n 4 -f --maxfail=1
 ```
 
+#### How to Execute the Tests?
+
+1. Make sure that you have `leveldb` and `tox` installed
+
+```sh
+brew install leveldb
+pip install tox
+```
+
+2. Execute `tox` for the full tests
+
+```sh
+tox -e py35-core # for Python 3.5
+tox -e py36-core # for Python 3.6
+```
```

Comment:
These are only running the core tests, not all the tests. (integration, ens, etc)

Maybe explicitly list another line with just `tox` separately that runs all the tests?